code/obj/item/clothing/gloves.dm

```diff
@@ -234,6 +234,16 @@ ABSTRACT_TYPE(/obj/item/clothing/gloves)
 		..()
 		setProperty("heatprot", 7)
 
+/obj/item/clothing/gloves/black/attackby(obj/item/W as obj, mob/user as mob)
```

Comment:
```suggestion
/obj/item/clothing/gloves/black/attackby(obj/item/W, mob/user)
```
Redundant and might even hurt performance (for however many fractions of a micro second)